README.md

```diff
@@ -1 +1,229 @@
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+# 📖 [Hello-Microverse] <a name="about-project"></a>
+
+
+**[Hello-Microverse]** is about Linters setup and ensuring all linters check pass, and also add & display a simple HTML text and basic css styling..
+
+## 🛠 Built With <a name="built-with"></a>
+-HTML
+
+-CSS
+### Tech Stack <a name="tech-stack"></a>
+
+
+<details>
+  <summary>Version Control</summary>
+  <ul>
+    <li><a href="https://github.com/">Git Hub</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Visual Studio Code</summary>
+  <ul>
+    <li><a href="https://code.visualstudio.com/">Visual Studio Code</a></li>
+  </ul>
+</details>
+
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+
+
+- **[Linters Setup]**
+- **[Follow HTML and CSS best practice]**
+- **[Display text]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+
+- [Live Demo Link](https://vikas924.github.io/Hello-Microverse/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+- Visual Studio Code.
+- Node JS.
+- Git bash.
+- GitHub Account.
+
+### Setup
+
+Clone this repository to your desired folder:
+
+Use git clone command or download ZIP folder
+
+```sh
+  cd my-folder
+  git@github.com:vikas924/Hello-Microverse.git
+```
+
+
+### Install
+
+Install this project with:
+
+npm
+
+```sh
+  cd my-project
+  npm init -y
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+npm start or live server
+
+
+```sh
+  GitHub Pages Server
+```
+
+### Run tests
+
+To run tests, run the following command:
+
+
+```sh
+ npx stylelint "**/*.{css,scss}"
+```
+
+
+### Deployment
+
+You can deploy this project using:
+
+GitHub Pages
+
+```sh
+ https://vikas924.github.io/Hello-Microverse/
+```
+ 
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+
+👤 **Vikas Chib**
+
+- GitHub: [@vikas924](https://github.com/vikas924)
+- Twitter: [@vikas410190](https://twitter.com/vikas410190)
+- LinkedIn: [Vikas Singh](https://www.linkedin.com/in/vikas-singh-586a07132/)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+> Describe 1 - 3 features you will add to the project.
+
+- [ ] **[new_feature_1]**
+- [ ] **[new_feature_2]**
+- [ ] **[new_feature_3]**
```

Comment:
- [ ] Nice job adding the changes requested by the previous reviewer. However, kingly add three features this project will likely improve on based on your personal discretion 👍  
- [ ] Secondly, please delete the comment on one `173` as it only serves as a guide to you. this keeps your code clean